Dockerfile

```diff
@@ -1,10 +1,7 @@
-FROM ghcr.io/deephaven/server:edge AS dx-server
+FROM ghcr.io/deephaven/server:${VERSION:-latest} AS dx-server
 COPY data/app.d /app.d
-HEALTHCHECK --interval=3s --retries=3 --timeout=11s CMD /bin/grpc_health_probe -addr=localhost:8080 -connect-timeout=10s || exit 1
-
-FROM ghcr.io/deephaven/web:edge AS dx-web
 COPY data/layouts /data/layouts
-RUN chown www-data:www-data /data/layouts
+HEALTHCHECK --interval=3s --retries=3 --timeout=11s CMD /bin/grpc_health_probe -addr=localhost:8080 -connect-timeout=10s || exit 1
```

Comment:
The healthcheck should be part of the jetty image, so we shouldn't need it here - and it would be port 10000 instead of 8080...